scripts/gitlab-build.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
-
 set -e # fail on any error
-set -u # treat unset variables as error
```

Comment:
Imho was pretty useful. Maybe set `$protected` to some default value in an if statement?

Reply:
temporary solution

Reply:
to what? can you elaborate?